index.html

```diff
@@ -211,24 +168,43 @@ <h2>Turn {{ turn }} - CPs <span class="badge badge-info">{{ constructionPoints }
         <div class="row">
             <div class="col">
                 <small>
-                    Created by Quique Porta (<a href="https://twitter.com/quiqueportac">@quiqueportac</a>).
+                    Originally created by Quique Porta (<a href="https://twitter.com/quiqueportac">@quiqueportac</a>).
                 </small>
             </div>
         </div>
         <div class="row">
             <div class="col">
                 <small>
-                    Source code on <a href="https://github.com/quiqueporta/space-empires-4x-app">Github</a>.
+                    Original source code on <a href="https://github.com/quiqueporta/space-empires-4x-app">Github</a>.
                 </small>
             </div>
         </div>
+        <div class="row">
+            <div class="col">
+                <small>
+                    Modified by Scott Lewis.
+                </small>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col">
+                <small>
+                    Scott's modified source code on <a href="https://github.com/sigmazero13/space-empires-4x-app">Github</a>.
+                </small>
+            </div>
+        </div>
+        <div class="row p-1">
+            <div class="col">
+                <small>Please, give <a href="mailto:quiqueporta@gmail.com">Quique</a> or <a href="mailto:sigmazero13@gmail.com">Scott</a> feedback.</small>
+            </div>
+        </div>
```

Comment:
Maybe we can create a button with a popup with all the contributors, more than add it to the bottom of the first tab.
What do you think?

Reply:
That works for me :)  I propose that be done in a future commit, though.